tests/flexmock_test.py

```diff
@@ -526,6 +527,14 @@ def method(self):
         assert_equal("instance", user.method())
         assert_equal("class", user2.method())
 
+    def test_should_call_with_class_default_attributes(self):
```

Comment:
I don't really understand the purpose of this test, could you add a docstring to this one?

Reply:
I added a docstring: https://github.com/flexmock/flexmock/pull/56/commits/d9e9e58c21ad3e705c90502e254a349e83ad8648

This test is for this line:
https://github.com/flexmock/flexmock/blob/3edc739258652d876a9f7d01775f658b9e6fbc03/src/flexmock/api.py#L141